R/mime.R

```diff
@@ -2,8 +2,13 @@
 # TODO: Currently outputs 8-bit quoted-printable, verify this is OK (Apple Mail seems to do it)
 
 # Turns a blastula email object into an RFC2822 message
-generate_rfc2822 <- function(eml, date = Sys.time(), subject = NULL,
-  from = NULL, to = NULL, cc = NULL) {
+generate_rfc2822 <- function(eml,
+                             date = Sys.time(),
+                             subject = NULL,
+                             from = NULL,
+                             to = NULL,
+                             cc = NULL,
+                             bcc = NULL) {
```

Comment:
I don't believe this is correct, the whole point of bcc is that it's blind, i.e. not included as part of the message.